### DISCUSSION TITLE
defaultOpen property not working as expected

### DISCUSSION THREAD

**Discussion by RiCatarino** [fuma-nama/fumadocs#541] [Q&A] [answered]
### Issue Description

When I include `"defaultOpen": false` in my configuration, it's not working as expected. The folder is always open regardless of the `defaultOpen` setting.

### Steps to Reproduce

1. Include `"defaultOpen": false` in the configuration.
2. Observe that the folder remains open.

### Expected Behavior

The folder should be closed when `"defaultOpen": false` is specified.

### Actual Behavior

The folder is always open, even when `"defaultOpen": false` is specified.

### Environment

- **fumadocs version:** 12.3.2

### Additional Information

![image](https://github.com/fuma-nama/fumadocs/assets/29524722/22eca364-a5ab-4137-b203-ce7c986a37db)

**Comment by fuma-nama** [comment, marked as answer]
This is expected, you have to adjust the default open level of sidebar, since the priority of sidebar options is higher.

See https://fumadocs.vercel.app/docs/ui/blocks/layout#reference-1

**Reply by RiCatarino** [reply]
Thank you. That was the solution.
